spacemacs/packages.el

```diff
@@ -2492,7 +2492,7 @@ displayed in the mode-line.")
 
 (defun spacemacs/init-volatile-highlights ()
   (use-package volatile-highlights
-    :init
+    :config
```

Comment:
Yeah, as noted above, it's not an eval-after-load, it's a require, so it does what you'd want.

In general, it's dangerous to reference package members in an `:init` form, because it depends on the autoloads already being set up.